Sources/App/Core/Ref.swift

```diff
@@ -44,16 +44,19 @@ extension Version: Referenceable {}
 ///                              |
 ///                              '-< Product
 /// ```
+// periphery:ignore
```

Comment:
As above, currently unused but keeping it around for when we write new queries. These are useful tools to have around.